src/Infrastructure/LeanCode.Serialization/JsonConverters/JsonStringEnumConverters.cs

```diff
@@ -1,34 +1,85 @@
+using System.Diagnostics.CodeAnalysis;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace LeanCode.Serialization;
 
+[RequiresDynamicCode(
```

Comment:
Since we have the generic variant now, is there a reason for keeping these (for our usecase)?

Reply:
Probably not, or at least I can't see one. I suppose I'll just remove them.

(Funny how they survived a total of 2 weeks :upside_down_face:)

Reply:
Yeah, since you're a fan of removing code, let's remove the code ;P